packages/terra-overlay/tests/wdio/overlay-spec.js

```diff
@@ -1,6 +1,163 @@
-/* global browser, Terra */
+/* global before, browser, Terra */
 
 describe('Overlay', () => {
+  describe('Default', () => {
+    beforeEach(() => browser.url('/#/tests/overlay-tests/default'));
+
+    Terra.should.beAccessible({ selector: '#default-overlay' });
+    Terra.should.matchScreenshot({ selector: '#default-overlay' });
+
+    it('Expect default to have full screen', () => {
+      expect(browser.getAttribute('#default-overlay', 'class')).contains('fullscreen');
+    });
+  });
+
+  describe('Overlay container', () => {
+    beforeEach(() => browser.url('/#/tests/overlay-tests/container'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+  });
+
+  describe('Scrollable Full Screen Overlay', () => {
+    beforeEach(() => browser.url('/#/tests/overlay-tests/fullscreen-scrollable'));
+
+    Terra.should.beAccessible({ selector: '#scrollable-overlay' });
+    Terra.should.matchScreenshot({ selector: '#scrollable-overlay' });
+  });
+
+  describe('Scrollable relative container overlay', () => {
+    beforeEach(() => browser.url('/#/tests/overlay-tests/container-scrollable'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+  });
+
+  describe('Overlay on Request-Close', () => {
+    describe('OnRequestClose- Full Screen Overlay', () => {
```

Comment:
These tests are nicely written good job!! My only request, can we get a before & after screenshot of the `on request close` overlays?

Reply:
Thanks @emilyrohrbough  :). Updated the test cases.